src/Core/Content/ProductExport/Service/ProductExportFileHandler.php

```diff
@@ -100,7 +100,7 @@ private function isCacheExpired(ExportBehavior $behavior, ProductExportEntity $p
 
     private function ensureDirectoryExists(): void
     {
-        if (!$this->fileSystem->fileExists($this->exportDirectory)) {
```

Comment:
Can we maybe remove that code completely. I think Flysystem does create the folders if they are missing when you are writing a file 🤔